src/client/bridge/gateway/shard_manager.rs

```diff
@@ -38,7 +38,7 @@ use crate::CacheAndHttp;
 /// Initialize a shard manager with a framework responsible for shards 0 through
 /// 2, of 5 total shards:
 ///
-/// ```rust,no_run
+/// ```rust
```

Comment:
Are we sure we want to run this? It would call `Http::get_gateway` for each check in CI; I'd like to not spam Discord needlessly.
```suggestion
/// ```rust,no_run
```

Reply:
That was an unintended regression.